### DISCUSSION TITLE
Should useAsync and LoadingView be displayed together?

### DISCUSSION THREAD

**Discussion by drewvolz** [StoDevX/AAO-React-Native#6116] [General]
We have duplicate loading UI now that useAsync is making its way to more views. Does it make sense to keep these two things together or should we opt to only show one of them?

**Comment by drewvolz** [comment]
I think the only issue is that refreshing is set to true on initial load. If we were to prevent this we would probably not run into this visual issue.

**Comment by hawkrives** [comment]
Resolved!
